src/extension/features/general/hide-referral-banner/index.css

```diff
@@ -1,3 +1,3 @@
-div.referral-program {
+div.sidebar-bottom {
```

Comment:
If you add the display: none here you'll also be hiding the sidebar collapse button. Leaving it on the div that contains class referral-program is needed and what the code already does. What am I missing? Can you provide a screenshot of the problem you're seeing?

Reply:
> If you add the `display: none` here you'll also be hiding the sidebar collapse button. Leaving it on the div that contains class referral-program is needed and what the code already does. What am I missing? Can you provide a screenshot of the problem you're seeing?

![image](https://user-images.githubusercontent.com/1592048/85591245-7038f100-b645-11ea-9452-13f4d1d8e67f.png)

I don't have any "collapse button" here.

Reply:
This is coming soon to a YNAB near you. Seriously, this is an upcoming change that hasn't been made available to everyone yet. There really is no need for a change in light of that. Here's a little screenshot;
![image](https://user-images.githubusercontent.com/11460228/85592316-ae192380-b60b-11ea-9295-117f2e91f51c.png)
I reckon you'll just have to trust us on this one. I do appreciate you pushing a PR as we need more contributors.